### ISSUE TITLE
Watch status is not synced from Trakt to Plex (but ratings are)

### ISSUE THREAD

**Issue by tonitonae** [Taxel/PlexTraktSync#567]
### The problem

_A bunch of movies that are watched and rated in Trakt do not sync watch status into Plex, although ratings are fetched from Trakt_

### Expected behavior

_Both watch status and ratings should be synced._

### Steps to reproduce the behavior

_Mark a movie present in Plex library as watched in Trakt. Rate it. Sync._

### Workarounds

_None that I am aware of._

### Environment

- Version: 0.13.26
- Python version: 3.8.11
- Platform: Linux 4.4.59+ (on a Synology NAS)

Default config is being used:

```
{
    "log_debug_messages": false,
    "logging": {
        "append": false
    },
    "cache": {
        "path": "$PTS_CACHE_DIR/trakt_cache"
    },
    "sync": {
        "liked_lists": true,
        "watchlist": true,
        "watched_status": true,
        "collection": true,
        "ratings": true
    },
    "xbmc-providers": {
        "movies": "imdb",
        "shows": "tvdb"
    },
    "excluded-libraries": [
        "Private",
        "Family Holidays"
    ]
}
```

Thanks!

**Comment by twolaw** [comment]
Update to last version (0.15.6 at the moment) and try again.

**Comment by tonitonae** [comment]
Thanks for your suggestion.

Updated to 0.15.6 but still the same behavior (no update from trakt to plex).

Thanks!

**Comment by fnwc** [comment]
I was having the token error, which I fixed by creating a another PlexTraktSync account. I can now sync, but none of the watched status syncs appear to be pushing to Trakt.

**Comment by twolaw** [comment]
> Thanks for your suggestion.
> 
> Updated to 0.15.6 but still the same behavior (no update from trakt to plex).


Then edit the first post please and show your new config file.
Did you update python requirements ?
Can you show logs.

**Comment by tonitonae** [comment]
Hi all.

I have had time for a more thorough review of the data and I have found out that the check-ins for those movies have disappeared from Trakt. I do not know when and how this happened (maybe it is the Trakt plugin for Kodi to blame?) but it is normal then that watch status did not sync, as it was not present in Trakt.

I have now checked in some of those movies manually in Trakt and watched status syncs to Plex. Then batch set as watched in Plex the remaining movies and they synced perfectly to Trakt, so no problem at all.

I apologise for the noise. I should have double-checked that those movies were watched in Trakt (actually they were, I am sure of that, but I can not tell when those check-ins disappeared).

I think this issue can be closed now. I just leave this message here as a remainder for everybody to double-check before posting.

Cheers!
